README_NASA_IMPACT.md

```diff
@@ -0,0 +1,26 @@
+# Titiler NASA IMPACT fork Readme
+
+```
+export TITILER_VERSION=0.3.5 # whichever tag you want to merge from
+git clone git@github.com:NASA-IMPACT/titiler.git
+git remote add upstream git@github.com:developmentseed/titiler.git
+git fetch upstream
+```
+
+```
```

Comment:
Nit but maybe add an explanation as to why these 2 code blocks are distinct.

Perhaps above the first:
> Fetch code from appropriate version of Titiler.

And the second:
> Merge updates into mater branch of the NASA-IMPACT fork.

Reply:
I actually forgot to add any explanatory text, oops. Added now.